### PR TITLE
Release 1.40.0

### PR DESCRIPTION
**Pull request by psss**
### Follow the steps below to create a new major or minor release:

- [x] Update overview.rst with new contributors since the last release
- [x] Review the release notes in releases.rst, update as needed
- [x]  Add a Release x.y.z commit, empty if needed: git commit --allow-empty -m "Release x.y.z"
- [x]  Create a pull request with the commit, ensure tests pass, merge it

### Create a new github release based on the tag above

- [ ] Move the fedora branch to point to the new release
- [ ] Tag the commit with x.y.z, push tags git push --tags
- [ ]  Create a new github release based on the tag above
- [ ]  Mention the most important changes in the name, do not include version
- [ ] Use ; as a delimiter, when multiple items are mentioned in the name
- [ ] Push the “Generate release notes” button to create the content
- [ ] Prepend the “See the release notes for the list of interesting changes.” line
- [ ] Publish the release, check Fedora pull requests, make sure tests pass and merge

### Finally, if everything went well

- [ ] Close the corresponding release milestone
- [ ] Once the non development copr build is completed, move the quay branch to point to the release commit as well to build fresh container images.

### Handle manually what did not went well

- [ ] If the automation triggered by publishing the new github release was not successful, publish the fresh code to the pypi repository manually using make wheel && make upload
- [ ] If there was a problem with creating Fedora pull requests, you can trigger them manually using /packit propose-downstream in any open issue.

### CODE REVIEW

File: docs/guide.rst [hunk 1]
```diff
@@ -878,7 +878,7 @@ copied to the guest.
 Conditional step configuration
 ------------------------------
 
-.. versionadded:: 1.39
+.. versionadded:: 1.40
```
Comment (martinhoyer) [inline docs/guide.rst:881]: Shouldn't these be 1.40.0 as well? Or would it break something? (not blocking release PR, just noticed it here)

**Review by martinhoyer — APPROVED** [review]

File: docs/guide.rst (hunk 1 above)
Comment (psss) [reply docs/guide.rst:881]: Hm, good question. We're not adding new features in micro versions so I'd say it's ok. But if we want to use `x.y.z` versions across the whole documentation consistently, let's do that in a separate pull request and cover all occurences.

**Comment by psss** [comment]
There are some orange plans in the [provision job](https://artifacts.osci.redhat.com/testing-farm/dcff5eb7-5c6c-4cbf-b189-c42a305aeeb2/), looks unrelated but let's give it one more try.

/packit test --id provision

**Comment by psss** [comment]
Let's see, one of them must be green enough ;-)

* [one](https://artifacts.osci.redhat.com/testing-farm/dcff5eb7-5c6c-4cbf-b189-c42a305aeeb2/)
* [two](https://artifacts.osci.redhat.com/testing-farm/6e43792d-fdb8-462a-820e-afc550067c1a/)
* [three](https://artifacts.osci.redhat.com/testing-farm/741b7c43-fa76-4523-9a6a-634b452d9b21/)

**Comment by psss** [comment]
Everything green in [6e43792d](https://artifacts.osci.redhat.com/testing-farm/6e43792d-fdb8-462a-820e-afc550067c1a/) except for `Failed to boot testcloud instance` errors in `/tests/multihost/web` but that test is green in [741b7c43](https://artifacts.osci.redhat.com/testing-farm/741b7c43-fa76-4523-9a6a-634b452d9b21/). Merging.

**Pull request merged**